integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java

```diff
@@ -49,6 +51,27 @@ public static long getGid(String userName) {
     return getIdInfo("-g", userName);
   }
 
+  /**
+   * Retrieves the gid of the given user name and group name.
+   * The group must be registered and must contain the user to get the gid.
+   *
+   * @param userName the user name
+   * @param groupName the group name
+   * @return gid
+   */
+  public static long getGid(String userName, String groupName) throws IOException {
```

Comment:
I think this implementation only works when `userName` is part of `groupName`, but that may not be the case. [This answer](https://stackoverflow.com/a/10910180/401884) gives a solution for both osx and linux, and we have util methods for determining whether we're on OSX or Linux in `OSUtils`

Reply:
Great advice!! I have changed the two methods to make use of different commands in OS and Linux. PTAL, thanks!